admin/email/email.go

```diff
@@ -63,14 +65,20 @@ func (c *Client) SendOrganizationInvite(opts *OrganizationInvite) error {
 		opts.InvitedByName = "Rill"
 	}
 
+	queryParams := url.Values{}
+	// Add query parameters
+	queryParams.Add("redirect", mustJoinURLPath(c.frontendURL, opts.OrgName))
+	// Construct the final URL with query parameters
+	finalURL := c.externalURL + "/auth/signup" + "?" + queryParams.Encode()
```

Comment:
1. Use `mustJoinURLPath` to join the finalURL
2. nit: better to have a block comment at the top explaining the high-level output, like:
```go
// Create link URL as "{{ admin URL }}/auth/signup?redirect={{ org frontend URL }}"
queryParams := url.Values{}
queryParams.Add("redirect", mustJoinURLPath(c.frontendURL, opts.OrgName))
finalURL := mustJoinURLPath(c.externalURL, "/auth/signup") + "?" + queryParams.Encode()
```